physics/README.md

```diff
@@ -0,0 +1,4 @@
+# NOTE
+
+All the examples in this subdirectory are not useful past 2019 until a new version of the robotpy simulator is released.
```

Comment:
Uh, I thought we already released a version of pyfrc which adds back in the physics module. These just need to actually be updated.